build.cake

```diff
@@ -337,19 +360,25 @@ Task("generate-mapping")
 	if (!FileExists(supportJson))
 		DownloadFile(SUPPORT_CONFIG_URL, supportJson);
 
-	var reader = new StreamReader("./mappings/androidx-mapping.csv");
-	var csv = new CsvReader(reader);
+	// read the newly generated mapping file
+	var csvReader = new StreamReader("./mappings/androidx-mapping.csv");
+	var csv = new CsvReader(csvReader);
 	var records = csv.GetRecords<dynamic>()
 		.Cast<IDictionary<string, object>>()
 		.Select(r => $"{r["Support .NET assembly"]}|{r["AndroidX .NET assembly"]}")
+		.Union(new [] {
+			"Xamarin.Android.Support.v4|Xamarin.AndroidX.Legacy.Support.V4",
+			"Xamarin.Android.Support.MultiDex|Xamarin.AndroidX.MultiDex",
```

Comment:
These two packages are special, the v4 actually has no types, so e have to manually add them. The MultiDex are not actually a thing for Support and not directly used in AndroidX.